CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
 - \#2662 - Change Header When Applying Filter in App Collection View
 - \#2663 - Adapt filter count, reflect current result set
 - \#2702 - Show proper info on app creation auth error
+- \#2704 - Change format of groupId under app name
+- \#2705 - Link groupId in search results to individual groups
```

Comment:
This could be also seen as a feature that would belong under _Added_.
Not sure..